docs/concepts/lite-agent.mdx

```diff
@@ -140,58 +145,53 @@ class MarketResearchFlow(Flow[MarketResearchState]):
         Format your response according to the specified structure.
         """
 
-        # Execute the analysis
-        result = await analyst.kickoff_async(query)
-        self.state.analysis = result.pydantic
-        return result.pydantic
+        # Execute the analysis with structured output format
+        result = await analyst.kickoff_async(query, response_format=MarketAnalysis)
+        if result.pydantic:
+            print("result", result.pydantic)
+        else:
+            print("result", result)
+
+        # Return the analysis to update the state
+        return {"analysis": result.pydantic}
```

Comment:
`result.pydantic` might be None, isn't? that's why you've added `else` condition

Just a few suggestions to keep it clear
```
outcome = result.pydantic or result.raw
return {"analysis": outcome}
```